hosts_example.yml

```diff
@@ -141,6 +141,46 @@ all:
     #       file: "path/to/file.txt"
     #       topics: "test_topic"
 
+    #### Configuring Role Based Access Control ####
```

Comment:
@amitkgupta Can you review this documentation/ these variables?
At first, I thought they should be standardized, but looking at them now, they seem fine. 

The only ones that are not too good are token_services_public_pem_file and token_services_private_pem_file
But I'm content leaving them since we are very close to done

@JumaX wdyt about the vars?